src/containers/Admin/EditTab/BikeSearch/index.tsx

```diff
@@ -1,40 +1,56 @@
 import React, { useState, useEffect } from 'react'
 
-import { Coordinates, BikeRentalStation } from '@entur/sdk'
+import { Coordinates } from '@entur/sdk'
 import { Dropdown } from '@entur/dropdown'
+import { Station } from '@entur/sdk/lib/mobility/types'
 
 import service from '../../../../service'
 
 import './styles.scss'
+import { getTranslation } from '../../../../utils'
+
+const MAX_SEARCH_RANGE = 100_000
 
 interface Item {
     value: string
     label: string
 }
 
-function mapFeaturesToItems(features: BikeRentalStation[]): Item[] {
+function mapFeaturesToItems(features: Station[]): Item[] {
     return features.map(({ id, name }) => ({
         value: id,
-        label: name,
+        label: getTranslation(name) || id,
     }))
 }
 
 const BikePanelSearch = ({ onSelected, position }: Props): JSX.Element => {
-    const [stations, setStations] = useState<BikeRentalStation[]>([])
+    const [stations, setStations] = useState<Station[]>([])
 
     useEffect(() => {
-        let isMounted = true
+        const controller = new AbortController()
         if (position) {
-            service
-                .getBikeRentalStationsByPosition(position, 100000)
+            service.mobility
+                .getStations(
+                    {
+                        lat: position.latitude,
+                        lon: position.longitude,
+                        range: MAX_SEARCH_RANGE,
+                    },
+                    {
+                        signal: controller.signal,
+                    },
+                )
                 .then((data) => {
-                    if (isMounted) {
-                        setStations(data)
+                    setStations(data)
+                })
```

Comment:
Dette blir litt subjektivt, men 
```
.then((data) => {
    setStations(data)
})
```
kan skrives som 
```
.then(setStations)
```
når forrige steg og funksjonen tar inn 1 argument 😃

Reply:
Godt sett!:) Jeg valgte å ikke "shorthande" dette siden jeg var litt forvirret av den skrivemåten da jeg skulle sette meg inn i kodebasen. Tenkte derfor at det kanskje er litt mer lesbart slik, for evnt. nye utviklere som kommer inn:) Men det er jo subjektivt som du sier, og mulig det er mer irriterende om det er inkonsistent, så kan godt endre det hvis det er ønskelig.

Reply:
Lar dette ligge siden det ikke kommer noen sterke meninger på banen.